src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,126 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int GROW_FACTOR = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Object[] entries;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        entries = new Object[INITIAL_CAPACITY];
+        threshold = (int)(INITIAL_CAPACITY * LOAD_FACTOR);
```

Comment:
```suggestion
        threshold = (int) INITIAL_CAPACITY * LOAD_FACTOR;
```

Reply:
I got an error when i changed this line as you suggested. Incompatible types. Found: 'float', required: 'int'